libs/cardano-ledger-pretty/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## 1.2.1.0
 
 * Added `PrettyA` instance for `DRep`
+* Replace `ConwayTallyPredFailure era` with `ConwayGovPredFailure era`
+* Replace `ConwayTallyState era` with `ConwayGovState era`
+* Replace `ConwayVDelPredFailure era` with `ConwayGovCertPredFailure era`
```

Comment:
A new minor version is needed `1.2.3.0` that bumps up the bound on `cardano-ledger-conway ^>= 1.7`

Reply:
I think this should've been a major bump to begin with, since this renaming will break stuff.